thirdparty/overview.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2021-09-27T13:20:07.400Z" agent="5.0 (X11)" etag="_2Fxpr5zouTNdBK6z77W" version="15.3.5"><diagram id="pt4pjNS3_0Fx-sI1uwvt" name="Page-1">7Vttc9o4EP41zNx94Mbv0I8hSXsf7m46k+ld+lFYAisVlivLgPvrb2XL2AY5pQUDoUwmE++jt9Xug3ZXJgP3frH+IFAS/c0xYQPHwuuB+zBwHNvxx/BHIblGAs8qkbmgWGM18ES/EQ1W3TKKSdrqKDlnkiZtMORxTELZwpAQfNXuNuOsvWqC5mQHeAoR20X/o1hGJTr2rRr/k9B5VK1sW7plgarOGkgjhPmqAbmPA/decC7Lp8X6njBlvcou5bj3Ha0bxQSJ5T4D/glQFrx8WsfTf4cT6gnx5DwPK+WWiGV6x1pbmVcmEDyLMVGzWAN3soqoJE8JClXrCrwOWCQXDCRbPaIwygT5oPAHD4CE01gS8bgENVM9x4wyds8ZF8UC7vuR+gE8lYJ/IVVLzGNYY6IVJEKSdefW7Y1BgYqEL4gUOXTRrBvpXa4MHowa3vMDDSLNmvlmqtqw8KBt+wN2tg1mDhgsO8F0qRZkdB4XDcHXTDFiEhJltlqGp7n+W4ybcdh7009VR9UwTIsP0h10cLxkvTvLgr8gxnkCm5ERFRhsKfNqagDL2dsrAlwo20bPo381y1QYtLwpf1P+pvxN+ZvyR1G+Mx1Q8f4Y6cAhEV4PqKJ5lWYaIr7vGiK+8663iG/tWI5gSC21qJObMBPLjS3biRYXMuJzHiP2lwrUZZcXImWus2SUSd62NllT+dx4/qym+sPX0sNaz1wIeSXEsNvnptAYpcR6WCHV4/Cdyq9B5AkBQkP2BkmE1s32yi5NMeWZCMlHIihYmAhFGhrPoXUMjTB2TmRHI0ZptLFSaVZly9e5AqYvFnzFR64uKIq1X+nnm7knCEOSLtt6mIhUDAVrobzRofgkpI2ZPyqgpnTgtDldlUQ1KcsZa4puVPt51jrudbO2xaRTUvgMrO06Mnumrd2mreucgLZuZ3m1d+y0TbFTJkOU0GG6DDtjZu/RcatYxoiMZ+Fgu1iGliAck+nsOPE08Ntnj2uIp5YhnvZXQO/eS1zkwXQNB4x3jHNj92AItuLZaIsrenvlqJouP3wAWVvrjE9wANnjt0HPXzluenvGTXt0EXHz3Qlo6/UUN0Mep+CfIRcohPG/dOx0rXPHTr/Ty+WNxHedbPsmJyO8oLEhP3pzlwfbkcl0e7DBTuKxoCePhSHYXw4Z51+y5Nrc5njndptt71julgVcWBYw2jMLCM6SBFhnKJ5t/8baa2Gt7V5C7ur5p6BtcCm0Hd14ezBvj3LncOhxW6XlvfJ21FPNBWyNXr2p7HoNSCvgN55Iqj4PsMYQfjFJGM+Bgo41zQt5hjImf2+sQM9XzM1mMyc0FnM4mAZ+0E8xZ3qxeNJibnxoaQC7N7DngcCpgkkcUpLuWRSAUWXbjcbvbzXdpqHqFbh+8e1OlItoiNidblhQjNUyRt60T/FTVRsjp80E23Al7hmI4PRWbHS/9DioSFzk6Vc2RGxfGryJ2tA7e0lvd9+1HeQuQTBNr81d/vlL+e4wfYRPV+f96Jv3nGe47Tyt56o9HNtz6iuyNKQJUhdoNJXXdn3m2v15DsT6i+5lIl3/v4D7+D8=</diagram></mxfile>
```

Comment:
Can you not export the SVG with embedded draw.io data (as per the norm) instead of having two files?

Reply:
Fixed